Framework/Algorithms/src/CorrectKiKf.cpp

```diff
@@ -191,31 +191,17 @@ void CorrectKiKf::execEvent() {
   g_log.information("Processing event workspace");
 
   const MatrixWorkspace_const_sptr matrixInputWS =
-      this->getProperty("InputWorkspace");
-  EventWorkspace_const_sptr inputWS =
+      getProperty("InputWorkspace");
+  auto inputWS =
```

Comment:
It's not part of your changes, but it's confusing to have a local variable called `inputWS` with the same name as the member variable `this->inputWS`